EIPS/eip-4959.md

```diff
@@ -0,0 +1,325 @@
+---
+eip: 4959
+title: Standardize Ethereum Drink Menu
+description: A list of official Ethereum drinks and cocktails.
+author: Sam Wilson (@SamWilsn), Carson Wilcox (@codeofcarson)
+discussions-to: https://ethereum-magicians.org/t/eip-4959-standardize-ethereum-drink-menu/8761
+status: Draft
+type: Informational
+created: 2022-04-01
+---
+
+## Abstract
+
+Introduces a list of both alcoholic beverages and instructions for their creation, including the:
+
+ * Blocktail
+ * Dappiri <!-- Daiquiri -->
+ * Kate Sour <!-- Kyoto Sour -->
+ * Merkle Patricia <!-- Mary Pickford -->
+ * Proof of Steak <!-- Bloody Mary -->
+ * Proposer's Punch <!-- Planter's Punch -->
+ * Salty Dag <!-- Salty Dog -->
+ * Sex on the Beacon(chain) <!-- Sex on the Beach -->
+ * Sidechain <!-- Sidecar -->
+ * SLOAD Fashioned <!-- Old Fashioned -->
+ * Ramos DeFizz <!-- Ramos Fizz -->
+ * Rusty Node <!-- Rusty Nail -->
+
+## Motivation
+
+At conferences and events related to the Ethereum ecosystem, alcoholic beverages are frequently available for purchase, either through the official venue or from other vendors in close proximity. These vendors commonly attempt to capitalize on special events by adding new drink options to their menus. Often, these new drink options are named by combining an existing drink with a word or name from the lexicon of the special event. Since these vendors are often outsiders of the Ethereum community, the names only tangentially relate and are the source of much chagrin for Ethereum insiders.
+
+This EIP collects and standardizes a set of beverages for these vendors that is well-grounded in Ethereum lore.
+
+## Specification
+
+This proposal specifies twelve beverages, and two prepared ingredients.
+
+### Prepared Ingredients
+
+These ingredients may be prepared ahead of time, and are used in the drink recipes below.
+
+#### Earl Grey Tea Syrup
+
+##### Ingredients
+
+ * **`250 ml`** water
+ * **`250 ml`** sugar
+ * **`25 ml`** Earl Grey tea leaves
+
+##### Preparation
+
+1. In a saucepan, bring water just to a boil and remove from heat.
```

Comment:
The numbered lists are all typod.

Reply:
Markdown automatically numbers the lists when rendered, and this style lets you add/remove items without manually renumbering.

Reply:
Ah, then maybe replace the ones that were manually numbered 2, 3, etc?